src/app/contextBroker/contextBroker.cpp

```diff
@@ -719,7 +721,9 @@ static int loadFile(char* path, char* out, int outSize)
     return -1;
```

Comment:
Suggested by @kzangeli through Skype
```suggestion
    return NULL;
```